app/helpers/application_helper.rb

```diff
@@ -337,21 +337,11 @@ def impersonating?
   end
 
   def saas?
-    ThreeScale.tenant_mode.multitenant?
-  end
-
-  def major_and_minor_version_path
-    info = System::Deploy.info
-
-    if saas?
-      "red_hat_3scale/#{info.major_version}-saas"
-    else
-      "red_hat_3scale_api_management/#{info.major_version}.#{info.minor_version}"
-    end
+    !ThreeScale.config.onpremises
```

Comment:
In SaaS, `onpremises` is missing (so, is `nil`), and `tenant_mode: multitenant`.

In on-premises (including RHOAM), `onpremises: true` and `tenant_mode: <%= ENV['TENANT_MODE'] %>`, whose values can be `master`, `provider` and `developer`.

I think it's confusing to check for `multitenant` tenant mode, when we have `onpremises` property.